c/tests/test_tables.c

```diff
@@ -145,60 +145,62 @@ test_table_collection_equivalence(void)
     CU_ASSERT_FATAL(ret >= 0);
 
     // Equality of empty vs non-empty
-    ret = tsk_table_collection_equivalence(
+    ret = tsk_table_collection_equivalent(
         &tc1, &tc2, TSK_EQ_CMP_METADATA | TSK_EQ_CMP_PROVENANCE);
     CU_ASSERT_FALSE(ret);
     // Copying over tc1 to tc2
     ret = tsk_table_collection_copy(&tc1, &tc2, 0);
     CU_ASSERT_EQUAL_FATAL(ret, 0);
 
-    // Equivalence for different metadata/schema
+    // equivalent for different metadata/schema
```

Comment:
do you mean "equivalent *except* for metadata?

Reply:
yes!